Documentation/Localization.ru_RU/ListModule/UsingEffectively/Index.rst

```diff
@@ -0,0 +1,63 @@
+.. include:: /Includes.rst.txt
+.. _using-the-list-module-effectively:
+
+========
+Введение
+========
+
+Модуль :guilabel:`Веб > Список` / :guilabel:`Web > List` позволяет перемещаться по страницам или папкам сайта и просматривать все записи, хранящиеся на них. Модуль :guilabel:`List` также позволяет создавать и управлять записями, для которых нет специального модуля.
+
+Примерами различных типов записей, которые могут храниться на данной странице, являются подстраницы, элементы содержимого и новостные сообщения.
+
+==================================
+Использование модуля Список / List
+==================================
+
+Выберите модуль :guilabel:`Веб > Список` / :guilabel:`Web > List` и перейдите на страницу "Congratulations". Вы должны увидеть следующее:
+
+.. include:: /Images/AutomaticScreenshots/ListModule/ListModule.rst.txt
+
+В режиме списка отображается список записей, хранящихся на текущей странице, сгруппированных по типу. Название, которое отображается для каждой записи, зависит от того, какое поле используется в качестве метки. Для "pages" (страниц) этим полем является "Pagetitle".
+
+В режиме списка в **заголовке документа** предлагается несколько пиктограмм различных действий:
+
+1. **Создать** / **Create** новую запись (появится мастер, позволяющий выбрать тип записи).
```

Comment:
Are these texts that are displayed in the Backend? I would then use :guilabel:`Create` to make this clear

Reply:
In order not to do a lot of work with pictures (replacing with Russian interface), in translation I do it like this. Besides, it is not known where the editors will work - in the interface with Russian translation, or with English. And this way, in my opinion, it will be clear to the editors.

Reply:
That is, if the interface in their backend is in Russian, they might get confused..... And so - there is both Russian translation of interface labels, and original English, corresponding to the screenshot.

Reply:
By using double stars the text will be printed in **bolt**. If you use ``` :guilabel:`Create` ``` instead the text will be marked as a gui label in screen readers and will be displayed in a different layout. See also https://docs.typo3.org/m/typo3/docs-how-to-document/main/en-us/WritingReST/Reference/Code/InlineCode.html

Reply:
I see what you mean....